config/samples/httpbin_upstream_certmanager_cert.yaml

```diff
@@ -0,0 +1,36 @@
+##### How to use upstream certificate refs:
+
+#
+# First apply this manifest using the command
+# "kubectl apply -f config/samples/httpbin_upstream_certmanager_cert.yaml -n tyk"
+#
+# The operator will try to create the ApiDefinition and will succeed but will log an error that a certificate is missing
```

Comment:
Is there any reason why we should first create this ApiDefinition instead of the certificate? If not, maybe we can say something like "Operator can work even if the certificate is not in the cluster."

Reply:
No reason - it will work just fine if the secret is created first.  Will amend and add a statement as you suggest

Reply:
Actually I will change the order to create the certificate first, but also mention that it can work the other way around

Reply:
All done.  Ready to review again